lib/components/common/HoveredOptionContent.tsx

```diff
@@ -1,84 +1,84 @@
-import { Stack, Typography, Box } from '@mui/material';
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them